cmd/edv-rest/startcmd/start.go

```diff
@@ -94,14 +106,25 @@ var errInvalidDatabaseType = fmt.Errorf("database type not set to a valid type."
 
 var errCreateConfigStore = "failed to create data vault configuration store: %w"
 
+// nolint:gochecknoglobals
```

Comment:
Reason for the `nolint`?

Reply:
What I meant was: is there an alternative approach that doesn't require global variables?

Reply:
Well, I guess it's ok if the only alternative is literally a method that just returns that value. Someone else can weigh in here